configs/python/backend/taste/taste.yara

```diff
@@ -588,6 +588,7 @@ rule javascript_file {
         type = "script"
     strings:
         $var = { 76 61 72 20 } // var
+        $let = { 6C 65 74 20 } // let
```

Comment:
This config can be removed as far as I can tell. We don't use this when running in Docker, only `build/configs/taste.yara`. Can we remove this in the future?

Reply:
We could, I think I left it in case we merged from target/strelka, but we'll end up with plenty of conflicts if we do so anyways.